app/src/main/java/com/startup/go4lunch/api/OverpassRestaurantApi.java

```diff
@@ -0,0 +1,78 @@
+package com.startup.go4lunch.api;
+
+import android.location.Location;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.MutableLiveData;
+
+import com.startup.go4lunch.model.OverpassElements;
+import com.startup.go4lunch.model.OverpassGsonObject;
+import com.startup.go4lunch.model.Restaurant;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class OverpassRestaurantApi implements RestaurantApi, Callback<OverpassGsonObject> {
+
+    MutableLiveData<List<Restaurant>> restaurantListLiveData = new MutableLiveData<>();
+
+    public static Retrofit retrofit = new Retrofit.Builder()
```

Comment:
```suggestion
    private Retrofit retrofit = new Retrofit.Builder()
```
Optionnellement static